.github/workflows/autolink_jira.yaml

```diff
@@ -13,7 +13,8 @@ jobs:
   autolink:
     name: Add Jira Link to PR Title/Description
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'pull_request' && !startsWith(github.event.pull_request.title, '[A-Za-z\d-_.\\/]+') }}
+    if: ${{ github.event_name == 'pull_request' && !startsWith(github.event.pull_request.title, '([A-Za-z]{2,3})-(\d{1,5})') }}
```

Comment:
Will it never add Jira link to PR desc?

Reply:
Only when the jira tag *doesn't* exist on the title, i.e it's never prepended the jira link